plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/properties/GroupPropertySection.java

```diff
@@ -67,8 +68,8 @@ public void refresh() {
 			commandStack = null;
 			nameText.setText(getType().getName() != null ? getType().getName() : ""); //$NON-NLS-1$
 			commentText.setText(getType().getComment() != null ? getType().getComment() : ""); //$NON-NLS-1$
-			heightText.setText(Integer.toString(getType().getHeight()));
-			widthText.setText(Integer.toString(getType().getWidth()));
+			heightText.setText(Integer.toString(CoordinateConverter.INSTANCE.iec61499ToScreen(getType().getHeight())));
+			widthText.setText(Integer.toString(CoordinateConverter.INSTANCE.iec61499ToScreen(getType().getWidth())));
```

Comment:
Would it make sense for the user to enter IEC 61499 coordinates now?

Reply:
We thought about that. Currently the users sees the ruler and the grid in screen coordinates. Therefore we thought it makes more sense to have them also in the property sheet in screen coordinates.